### DISCUSSION TITLE
Add LCH support

### DISCUSSION THREAD

**Discussion by mkalamarz** [spatie/color#78] [Ideas]
LCH is a human-friendlier version of LAB
Wikipedia article: https://en.wikipedia.org/wiki/HCL_color_space
Formula: http://www.brucelindbloom.com/index.html?Eqn_Lab_to_LCH.html

**Comment by freekmurze** [comment]
That is currently not supported, but feel free to create a PR that adds this functionality. Make sure to update the readme and tests.
